src/manager.js

```diff
@@ -0,0 +1,100 @@
+import Cohort from './cohort.js'
+import { localeIncludes } from 'locale-includes'
+export default class Manager {
+  constructor() {
+    this.cohorts = []
+    this.cohortId = 0
+    this.studentId = 0
+  }
+
+  addCohort(name) {
+    if (name.length === 0) return 'Cohort needs to have name'
+    if (this.cohorts.some((cohort) => cohort.name === name))
+      return 'This cohort already exist'
+    this.cohortId++
+    this.cohorts.push(new Cohort(name, this.cohortId))
+  }
+
+  searchCohort(name) {
+    return this.cohorts.find((cohort) => cohort.name === name) || false
+  }
+
+  searchStudent(studentId) {
+    let student
+    this.cohorts.forEach((cohort) => {
+      if (typeof student !== 'object') {
+        student = cohort.students.find((student) => student.id === studentId)
+      }
+    })
+    return student || 'Student not found'
+  }
+
+  addStudent(cohortName, studentName, github, email) {
+    const currCohort = this.searchCohort(cohortName)
+    if (!currCohort) return 'Cohort Not Found'
+    if (this.studentChecker(studentName, github, email))
+      return 'Student already exists'
+    this.studentId++
+    return currCohort.addStudent(studentName, this.studentId, github, email)
+  }
```

Comment:
you have an addStudent method in both the Cohort and Manager class, can you delete one?